src/Discord.Net.Rest/DiscordRestApiClient.cs

```diff
@@ -71,6 +73,7 @@ internal void SetBaseUrl(string baseUrl)
             RestClient.SetHeader("accept", "*/*");
             RestClient.SetHeader("user-agent", UserAgent);
             RestClient.SetHeader("authorization", GetPrefixedToken(AuthTokenType, AuthToken));
+            RestClient.SetHeader("X-RateLimit-Precision", RateLimitPrecision.ToString().ToLower());
```

Comment:
As this conversion only occurs here at the moment, I am alright with it remaining as `ToString().ToLower()` - however, if in the future we decide (for some reason) to allow this on a per-request basis, we should definitely move to an extension method, a la `RateLimitPrecision.AsHeaderValue()`